scripts/main.js

```diff
@@ -71,4 +71,83 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+// ========================== OPDRACHT 1A ========================== //
+console.log('Opdracht 1a');
+console.log(`De afdeling Sales heeft ${departments.sales.numberOfEmployees} medewerkers`);
+console.log();
+// ========================== OPDRACHT 1B ========================== //
+console.log('Opdracht 1b');
+console.log(`Marketing is een leuke afdeling om te werken. ${departments.marketing.description}.`);
+console.log();
+// ========================== OPDRACHT 1C ========================== //
+console.log('Opdracht 1c');
+console.log(`De afdeling Customer Service heeft ${departments["customer-service"].numberOfEmployees} medewerkers`);
+console.log();
+// ========================== OPDRACHT 1D ========================== //
+console.log('Opdracht 1d');
+console.log(`Sales is een uitdagende afdeling om te werken als "Verkoopmanager". ${departments.sales.jobs[1].description}`);
+console.log();
+console.log();
+
+
+// ========================== OPDRACHT 2A, 2B en 2C ========================== //
+console.log('Opdracht 2a, 2b en 2c');
+const departmentChoice = prompt('Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]');
+
+switch (departmentChoice) {
+    case 'marketing':
+        // console.log(`Je koos ${departmentChoice}. ${departments.marketing.description}`);
+        console.log(`${departmentChoice.toUpperCase()} is een leuke afdeling om te werken. Er werken op dit moment ${departments[departmentChoice].numberOfEmployees} medewerkers.`)
+        break;
+    case 'sales':
+        // console.log(`Je koos ${departmentChoice}. ${departments.sales.description}`);
+        console.log(`${departmentChoice.toUpperCase()} is een leuke afdeling om te werken. Er werken op dit moment ${departments[departmentChoice].numberOfEmployees} medewerkers.`)
+        break;
+    case 'customer-service':
+        // console.log(`Je koos ${departmentChoice}. ${departments["customer-service"].description}`);
+        console.log(`${departmentChoice.toUpperCase()} is een leuke afdeling om te werken. Er werken op dit moment ${departments[departmentChoice].numberOfEmployees} medewerkers.`)
+        break;
```

Comment:
Dit zou je nog wat kunnen versimpelen
```suggestion
    case 'marketing':
    case 'sales':
    case 'customer-service':
        // console.log(`Je koos ${departmentChoice}. ${departments[departmentChoice].description}`);
        console.log(`${departmentChoice.toUpperCase()} is een leuke afdeling om te werken. Er werken op dit moment ${departments[departmentChoice].numberOfEmployees} medewerkers.`)
        break;
```